src/download-extensions

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# RPMs as operating system extensions, distinct from the base ostree commit/image
+# https://github.com/openshift/enhancements/blob/master/enhancements/rhcos/extensions.md
+
+import os
+import sys
+import yaml
+from cosalib import cmdlib
+
+destdir = sys.argv[1]
+tmpdir = 'tmp'
```

Comment:
nit: maybe do `tempfile.mkdir` here?

Reply:
In practice right now a whole bunch of things write hardcoded paths into `$COSA_DIR/tmp` (including bits of `upload-oscontainer` which is calling this).  Needs fixing a bit more holistically.